### DISCUSSION TITLE
versions

### DISCUSSION THREAD

**Discussion by EthanMcknney** [Mojang/minecraft-editor#189] [Feedback]
Can you add an update to the villagers like they do in Minecraft comes alive (MCA)?

**Comment by m-brisch** [comment]
Hello @EthanMcknney!

Since this is more of a general Minecraft request rather than one for Editor can you add your idea via the Minecraft feedback portal so it's picked up by the right team?

Here's the link: https://feedback.minecraft.net/hc/en-us

**Reply by EthanMcknney** [reply]
Thankyou
